docs/sources/setup-grafana/installation/windows/index.md

```diff
@@ -37,8 +37,8 @@ To run Grafana, open your browser and go to the Grafana port (http://localhost:3
 
 1. Extract this folder to anywhere you want Grafana to run from.
 
-1. Start Grafana by executing `grafana-server.exe`, located in the `bin` directory, preferably from the command line. If you want to run Grafana as a Windows service, then download
-   [NSSM](https://nssm.cc/). It is very easy to add Grafana as a Windows service using that tool.
+Start Grafana by executing `grafana-server.exe`, located in the `bin` directory, preferably from the command line. If you want to run Grafana as a Windows service, then download
```

Comment:
@DanCech , just a quick adjustment here. I'll create a separate PR for the Windows topic later.